sakai-evaluation-tool/src/webapp/component-templates/render_text_item.html

```diff
@@ -35,8 +35,10 @@ <h5 class="title">
 				<span rsf:id="itemText" class="label">Item text label.</span>
 
 				<!-- Previous
```

Comment:
OK, I found 3 additional commits in our 10.x branch, svn contrib r86864, r86865, r86870 (links in EVALSYS-951 JIRA).

The feature was originally ported from an even earlier evals branch, so our original merge to our 10.x msub evals made this error which was subsequently corrected.